thumbor/result_storages/__init__.py

```diff
@@ -12,6 +12,26 @@
 from os.path import exists
 from tornado.concurrent import return_future
 
+from thumbor.loaders import LoaderResult
+from thumbor.engines import BaseEngine
+
+class ResultStorageResult(LoaderResult):
+    def last_modified(self):
+        '''
+        Retrieves last_updated metadata if available
+        :return:
+        '''
+        return self.metadata['LastModified'] if 'LastUpdated' in self.metadata else None
+
+    def mime(self):
+        '''
+        Retrieves mime metadata if available
+        :return:
+        '''
+        return self.metadata['ContentType'] if 'ContentType' in self.metadata else BaseEngine.get_mimetype(self.buffer)
```

Comment:
Can just be `return self.metadata.get('ContentType', BaseEngine.get_mimetype(self.buffer))`